user/forgejo-runner/files/forgejo-runner

```diff
@@ -0,0 +1,7 @@
+type = process
+command = /usr/bin/forgejo-runner daemon
+working-dir = /var/lib/forgejo-runner/$1
```

Comment:
a) nothing creates this directory, so how does it work? i'm pretty sure dinit refuses to start a service with a nonexistent working-dir=

b) when do you ever run more than one of these at once with a separate workdir?

c) if you do run more than one, how do they share the same logfile= below? i am pretty sure that can't work

Reply:
a) It does not work without manual intervention by the user, but this service is only useful if the user has manually setup and registered the runner anyway. Requiring one more `mkdir` doesn't seem unreasonable to me. I don't really know how to provide documentation to a user about how to use a service that requires setup. Just comments in the service file?

b) You can create runners with different capabilities (this affects which runner `runs on` selects). You can also create runners for different contexts (global, organization, user, repo). It seems reasonable to me that someone may want more than one, but I'm often wrong. 
  
c) Fair point

d) I feel like its got a pretty generous amount of installation instructions in the main forgejo docs for something literally nobody should use, but I can understand why it may be undesirable to have in the repos. Maybe add `restricted`?